R/makeBarPolarPlot.R

```diff
@@ -0,0 +1,82 @@
+#' @keywords internal
+#' 
+#' @importFrom dplyr filter group_by summarize arrange desc pull mutate
+#' @importFrom rlang sym :=
+#' @importFrom ggplot2 ggplot aes geom_col ylim coord_polar theme_minimal 
+#'   theme element_blank labs geom_bar expand_limits element_text
+#' @importFrom cowplot draw_plot
+#' @importFrom grid unit
+#' 
+.makeBarPolarPlot <- function(df, idCol, metricCol, valueCol, 
+                              weightCol, scoreCol, methods, 
+                              ordering = "high-to-low") {
+    if (!(ordering %in% c("high-to-low", "low-to-high"))) {
+        stop("ordering must be 'high-to-low' or 'low-to-high'")
+    }
+    ## Define polar plots
+    rplots <- lapply(methods, function(m) {
+        ggplot2::ggplot(df %>% 
+                            dplyr::filter(!!rlang::sym(idCol) == m),
+                        ggplot2::aes(x = !!rlang::sym(metricCol), 
+                                     y = !!rlang::sym(valueCol),
+                                     fill = !!rlang::sym(metricCol))) + 
```

Comment:
Is this all replaceable by the `.data` pronoun from `rlang`? I am guessing it won't be really a difference in terms of efficiency, more just for the simplicity in reading it (call me old fashioned, but I'm still not too much into the newer bang-bang thingy...)